### PR TITLE
rgw: cleanup MIN macro with std::min

### PR DESCRIPTION
**Pull request by mikulely**

### CODE REVIEW

**Comment by mikulely** [comment]
Jenkins，retest this please.

**Comment by mikulely** [comment]
Jenkins，retest this please.

**Comment by mikulely** [comment]
@cbodley ping

File: src/rgw/rgw_rados.cc [hunk 1]
```diff
@@ -2074,12 +2074,12 @@ void RGWObjManifest::obj_iterator::operator++()
   if (ofs < head_size) {
     rule_iter = manifest->rules.begin();
     RGWObjManifestRule *rule = &rule_iter->second;
-    ofs = MIN(head_size, obj_size);
+    ofs = min(head_size, obj_size);
```
Comment (cbodley) [inline src/rgw/rgw_rados.cc]: can you please qualify these ones with `std::` also?

**Comment by mikulely** [comment]
Jenkins，retest this please.

**Comment by mikulely** [comment]
@cbodley done.

**Comment by yuriw** [comment]
[wip-yuri-testing-2017-10-18-1628](https://trello.com/c/CccCB9uk/182-wip-yuri-testing-2017-10-18-1628)

**Pull request merged**